runtimes/server-interface/chat.ts

```diff
@@ -35,8 +33,6 @@ export type Chat = {
     onTabChange: (handler: NotificationHandler<TabChangeParams>) => void
     onTabRemove: (handler: NotificationHandler<TabRemoveParams>) => void
     onVote: (handler: NotificationHandler<VoteParams>) => void
-    onCodeInsertToCursorPosition: (handler: NotificationHandler<InsertToCursorPositionParams>) => void
```

Comment:
Do you plan to reintroduce this later in the following PRs? I think `onCodeInsertToCursorPosition` will need to be sent to server at the end.